server/server.js

```diff
@@ -40,7 +40,7 @@ io.on("connection", (socket) => {
 
     console.log(`${userId} joined room ${roomId}`);
 
-    io.sockets.in(roomId).emit("userIsJoined", { users: users });
+    io.sockets.in(roomId).emit("userIsJoined", { users: data, success: true });
```

Comment:
These line of changes making the app crashed while clicking the button "peoples" in the room page. We have to emmit the data of all the users in the room to get the peoples list on each window.